src/providers/openai.ts

```diff
@@ -122,9 +122,9 @@ export class OpenAiEmbeddingProvider extends OpenAiGenericProvider {
     try {
       const embedding = data?.data?.[0]?.embedding;
       if (!embedding) {
-        throw new Error(`No embedding found in OpenAI embeddings API response: ${JSON.stringify(data)}`);
+        throw new Error('No embedding found in OpenAI embeddings API response');
```

Comment:
Relying on handling `catch` to print the `data`, otherwise it will be printed twice in the stack trace